### DISCUSSION TITLE
Show tooltip for map items on hover

### DISCUSSION THREAD

**Discussion by ghost** [AscensionGameDev/Intersect-Engine#1489] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not really a problem.

**Describe the solution you'd like**
Show item description as tooltip when hover on Inventory Items, Equipment Window Items, Spells. Etc. Basically if you move the cursor just a bit, the description will move with it. I have seen this style in many MMORPGs.

Example:

![picture](https://user-images.githubusercontent.com/72949583/97906004-4ee5c100-1d19-11eb-98b8-208a7c46efc0.png)

**Comment by lyimu** [comment]
Just curious if you mean hover on an item on the ground like this?
![image](https://user-images.githubusercontent.com/25600834/97894266-28a72d80-1ce7-11eb-917a-4c1d5e667305.png)
( **this is not a feature, this image is edited.** )

If so, I 100% agree!

**Comment by ghost** [comment]
> Just curious if you mean hover on an item on the ground like this?
> ![image](https://user-images.githubusercontent.com/25600834/97894266-28a72d80-1ce7-11eb-917a-4c1d5e667305.png)
> ( **this is not a feature, this image is edited.** )
> 
> If so, I 100% agree!

I forgot to specify, my bad, but I actually meant when hover on Inventory Items, Equipment Window Items, Spells. Etc. I didn't think of your idea, it sounds great but, what if there is more than one item on the same spot? I don't know how that could work :/

**Comment by Cheshire92** [comment]
I'm confused what the actual request is, because tooltips are already shown for items on the GUI.

**Comment by ghost** [comment]
> I'm confused what the actual request is, because tooltips are already shown for items on the GUI.

The item description window appears outside of the inventory and there is no way to make it appear starting from the cursor position once hovered on an item. Just as I showed in the main picture. Sorry, I can't explain better lol.

Here I found a perfect video example of what I mean:
http://www.youtube.com/watch?v=vXA6-p0wzcE&t=0m32s

**Comment by Cheshire92** [comment]
That wasn't what you had originally posted, hence the confusion.

**Comment by ghost** [comment]
> That wasn't what you had originally posted, hence the confusion.

I mean, it was, I just didn't explain it well. lol sorry :(

**Comment by WeylonSantana** [comment]
At the time this was created, it wasn't even thought about items below items, we can put an infinite number of items, all on the same tile, (example, when player dies and drops all items), how would that be in this case? Let's fill the screen with tooltip? shall we show only the first? or just the last one? I don't think it's a valid implementation
